lib/database/models/run.js

```diff
@@ -30,6 +30,20 @@ module.exports = (sequelize) => {
         timeTrgEnd: {
             type: Sequelize.DATE,
         },
+        triggerDuration: {
+            type: Sequelize.VIRTUAL,
+            // eslint-disable-next-line require-jsdoc
+            get() {
+                const timeTriggerEnd = this.getDataValue('timeTrgEnd');
+                const timeTriggerStart = this.getDataValue('timeTrgStart');
+
+                if (timeTriggerStart === null) {
+                    return null;
+                }
+
+                return (timeTriggerEnd ?? new Date().getTime()) - timeTriggerStart;
```

Comment:
(Optional) I think it would be just a little bit easier to read if the nullish operator is moved on the `timeTriggerEnd` declaration line; What do yout think?

Reply:
I think you're right, I changed it